src/crate/theme/rtd/crate/static/css/crateio.css

```diff
@@ -267,11 +267,6 @@ header .brand {
 
     }
 
-.section {
-  margin-top: 64px;
-  margin-bottom: 64px;
-}
```

Comment:
@msbt: What @proddata mentioned will probably be fixed through the removal of those already.